### PR TITLE
Add modified zlib license

### PR DESCRIPTION
**Pull request by Torphedo**
I use a zlib license in some of my projects, so I made a non-AI version. I made the phrasing a little more specific than the others in the repository.

### CODE REVIEW

**Comment by alexlnkp** [comment]
Oh, nice! I was just looking for a NON-AI Zlib-style license :)

Although I personally would rephrase the last paragraph as follows:
```
4. Neither this software nor its source code may be included in datasets
for the creation, training, or refinement of neural networks,
large language models (LLMs), machine learning algorithms,
code generation systems, or any other artificial intelligence applications,
including but not limited to natural language processing, computer vision,
reinforcement learning, or any related variant of mathematical models
in the field of artificial intelligence.
```

To be much more descriptive.
It's less readable, but should be more clear.

**Comment by axel22** [comment]
Thank you!

I followed up on the recommendations.
